.github/workflows/build_and_run_test.yml

```diff
@@ -15,20 +15,16 @@ jobs:
     runs-on: self-hosted
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout codes
+      uses: actions/checkout@v2
       with:
         submodules: true
-    - name: Add wasm toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-        target: wasm32-unknown-unknown
-        override: true
-    - name: Build relay node and parachain
+
+    - name: Build docker image
       run: |
-        cargo build --release
-        cd polkadot
-        cargo build --release
+        docker images
+        cd docker
+        ./build.sh
         cd ..
```

Comment:
Actually, I suspect `cd ..` can also be removed.

Reply:
> Actually, I suspect `cd ..` can also be removed.

could be..
I think I'll try `working-directory` keyword